### PR TITLE
Fix 176

### LINKED ISSUE #176: full chain - test call
The testcall at the moment only checks if the current website user typed in her phone number correctly into the database. It does not test if the full chain is actually working as in:

 - is the current website user is on shift today? (has led to problems in the past)
 - is the system is working in a way that Warning messages lead to calls to the shifter. 

I propose to make it very easy for the shifter to push a "dummy warning" into the system, via a button on the website. Whether this leads to a call or not is then up to the system.

### PR DESCRIPTION
**Pull request by dneise**
trying to fix #176 
by checking on the webinterface for recent "dummy alerts"

c.f. also 
https://github.com/fact-project/shifthelper_webinterface/pull/17

### CODE REVIEW

**Comment by dneise** [comment]
See also:
https://github.com/fact-project/shifthelper_deployment/pull/2

File: shifthelper/tools/__init__.py [hunk 1]
```diff
@@ -76,7 +76,23 @@ def fetch_users_awake():
...
+
+def fetch_dummy_alerts():
+    @retry(
+        stop_max_delay=30000,  # 30 seconds max
+        wait_exponential_multiplier=100,  # wait 2^i * 100 ms, on the i-th retry
+        wait_exponential_max=1000,  # but wait 1 second per try maximum
+        wrap_exception=True
+    )
+    def retry_fetch_fail_after_30sec():
+        return requests.get('http://webinterface/dummyAlert').json()
```
Comment (MaxNoe) [inline shifthelper/tools/__init__.py]: Maybe we should use the config file for these urls, as they can change (e.g. for local testing)

File: shifthelper/tools/__init__.py [hunk 2]
```diff
@@ -76,7 +76,23 @@ def fetch_users_awake():
         wrap_exception=True
     )
     def retry_fetch_fail_after_30sec():
-        return requests.get('https://ihp-pc41.ethz.ch/iAmAwake').json()
+        return requests.get('http://webinterface/iAmAwake').json()
```
Comment (MaxNoe) [inline shifthelper/tools/__init__.py]: Maybe we should configure this url in the config (keeping local testing in mind).

**Comment by dneise** [comment]
I take the webservice URLs now from the config file. 
adjusted the config file accordingly in a new branch which is now also used here:
see fact-project/shifthelper_deployment#2

I will test that deployment version now, in order to check for typos or so ... and then we can maybe deploy it.
Should I bump version numbers somewhere? If so where? In all 3 repos?

**Comment by dneise** [comment]
webinterface has no setup.py ... does it have a version?

**Comment by dneise** [comment]
Okay so this is fine now ... I merge it into the master

**Pull request merged**